velodrome/grafana-stack/dashboards/boskos-dashboard.json

```diff
@@ -749,6 +749,134 @@
       "showTitle": false,
       "title": "Dashboard Row",
       "titleSize": "h6"
+    },
+    {
```

Comment:
We don't deploy these JSON dashboard files, rather we periodically dump the config for the live Grafana dashboards here (see the [commit history](https://github.com/kubernetes/test-infra/commits/d416e2e9e3ce60376657039db7a3efad171f50bc/velodrome/grafana-stack/dashboards/boskos-dashboard.json)). We create new graphs by logging into Grafana and using the UI.
If we were to deploy this it would overwrite any changes made to the dashboard since Jan 28th.

I'm not sure what the best way for non-Googlers to add content to Velodrome is.
/cc @spiffxp

Reply:
Thanks for the context, I've removed the update from this PR, but we would definitely be interested in having a new dashboard row to track the aws-accounts added to Velodrome (after the metrics update included in this pr is merged and deployed).